node/impl/full/mpool.go

```diff
@@ -102,6 +103,7 @@ func (a *MpoolAPI) MpoolPushMessage(ctx context.Context, msg *types.Message) (*t
 		if b.LessThan(msg.Value) {
 			return nil, xerrors.Errorf("mpool push: not enough funds: %s < %s", b, msg.Value)
 		}
+		fmt.Println(msg.Cid())
```

Comment:
eh?

Reply:
Oop, that's definitely a diagnostic message...my bad, I'll kill it. 

Also you're quite Canadian, eh?